### PR TITLE
fix cmake bundle

### PR DESCRIPTION
**Pull request by paulo-coutinho**

### CODE REVIEW

File: ActiveRecord/Compiler/CMakeLists.txt [hunk 1]
```diff
@@ -21,5 +21,6 @@ install(
 	LIBRARY DESTINATION lib${LIB_SUFFIX}
 	ARCHIVE DESTINATION lib${LIB_SUFFIX}
 	RUNTIME DESTINATION bin
+	BUNDLE DESTINATION bin
 	INCLUDES DESTINATION include
```
Comment (SpaceIm) [inline ActiveRecord/Compiler/CMakeLists.txt:25]:
```suggestion
	DESTINATION bin
```

File: Encodings/Compiler/CMakeLists.txt [hunk 2]
```diff
@@ -15,5 +15,6 @@ install(
 	LIBRARY DESTINATION lib${LIB_SUFFIX}
 	ARCHIVE DESTINATION lib${LIB_SUFFIX}
 	RUNTIME DESTINATION bin
+	BUNDLE DESTINATION bin
 	INCLUDES DESTINATION include
```
Comment (SpaceIm) [inline Encodings/Compiler/CMakeLists.txt:19]:
```suggestion
	DESTINATION bin
```

File: PageCompiler/CMakeLists.txt [hunk 3]
```diff
@@ -23,5 +23,6 @@ install(
 	LIBRARY DESTINATION lib${LIB_SUFFIX}
 	ARCHIVE DESTINATION lib${LIB_SUFFIX}
 	RUNTIME DESTINATION bin
+	BUNDLE DESTINATION bin
 	INCLUDES DESTINATION include
```
Comment (SpaceIm) [inline PageCompiler/CMakeLists.txt:27]:
```suggestion
	DESTINATION bin
```

File: PocoDoc/CMakeLists.txt [hunk 4]
```diff
@@ -11,5 +11,6 @@ install(
 	LIBRARY DESTINATION lib${LIB_SUFFIX}
 	ARCHIVE DESTINATION lib${LIB_SUFFIX}
 	RUNTIME DESTINATION bin
+	BUNDLE DESTINATION bin
 	INCLUDES DESTINATION include
```
Comment (SpaceIm) [inline PocoDoc/CMakeLists.txt:15]:
```suggestion
	DESTINATION bin
```

**Review by SpaceIm — COMMENTED** [review]
simpler (when TARGETS in install() comands are executables only)

**Comment by paulo-coutinho** [comment]
Left something to this be merged?

**Comment by obiltschnig** [comment]
See e2f7e72a9

**Pull request closed without merging**